ray-operator/test/support/core.go

```diff
@@ -58,3 +59,13 @@ func storeContainerLog(t Test, namespace *corev1.Namespace, podName, containerNa
 	containerLogFileName := "pod-" + podName + "-" + containerName
 	WriteToOutputDir(t, containerLogFileName, Log, bytes)
 }
+
+func ExecPodCmd(t Test, pod *corev1.Pod, containerName string, cmd []string) {
+	kubectlCmd := []string{"exec", "-it", pod.Name, "-n", pod.Namespace, "-c", containerName, "--"}
```

Comment:
```suggestion
	kubectlCmd := []string{"exec", pod.Name, "-n", pod.Namespace, "-c", containerName, "--"}
```

I guess the -it is not necessary.